### PR TITLE
Move operator installation instruction

### PR DESCRIPTION
**Pull request by shamsher31**

### CODE REVIEW

**Review by jwendell — COMMENTED** [review]
I agree most of this is outdated, but I still use some of these commands as part of my developer workflow.

I think we should make it up-to-date and make it clear that these are intended for development purposes. Maybe we should rename it to DEVELOPERS.md and leave README high level?

**Comment by shamsher31** [comment]
> Maybe we should rename it to DEVELOPERS.md and leave README high level?

I think it's a good idea.

**Review by jwendell — COMMENTED** [review]
Just realized we already have DEVEL.md.

Not sure what to do. Perhaps point users to https://docs.openshift.com/container-platform/4.8/service_mesh/v2x/installing-ossm.html and developers to DEVEL.md?

**Comment by dgn** [comment]
> Just realized we already have DEVEL.md.
> 
> Not sure what to do. Perhaps point users to https://docs.openshift.com/container-platform/4.8/service_mesh/v2x/installing-ossm.html and developers to DEVEL.md?

sounds good to me. let's move this stuff to DEVEL.md

**Review by jwendell — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,139 +8,7 @@ This project is an operator that can be used to manage the installation of an [I
...
-### Uninstalling the Kiali Operator
-
-To uninstall the Kiali operator, execute the following command:
-
-```
-bash <(curl -L https://kiali.io/getLatestKialiOperator) --uninstall-mode true --operator-watch-namespace '**'
-```
-
-For more details on uninstalling the Kiali operator, see the [Kiali documentaton](https://www.kiali.io/documentation/getting-started/#_uninstall_kiali_operator_and_kiali).
+To install Red Hat OpenShift Service Mesh, first install the required Operators on OpenShift Container Platform and then create a `ServiceMeshControlPlane` resource to deploy the control plane. If you are an user, refer [documentation](https://docs.openshift.com/container-platform/4.8/service_mesh/v2x/installing-ossm.html) for how to install and configure Istio Operator on Red Hat OpenShift Service Mesh. If you are a developer, refer [DEVEL.md](DEVEL.md) for more advance configurations.
```
Comment (brian-avery) [inline README.md]:
```suggestion
To install Red Hat OpenShift Service Mesh, first install the required Operators on OpenShift Container Platform and then create a `ServiceMeshControlPlane` resource to deploy the control plane. If you are a user, refer to the [documentation](https://docs.openshift.com/container-platform/4.8/service_mesh/v2x/installing-ossm.html) for how to install and configure Istio Operator on Red Hat OpenShift Service Mesh. If you are a developer, refer [DEVEL.md](DEVEL.md) for more advance configurations.
```

**Review by dgn — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Pull request merged**